app/controllers/admin/getting_started_guide_controller.rb

```diff
@@ -19,6 +19,10 @@ def data
     has_sub_path = (path_parts.count == 2 && path_parts[1] != "/")
     sub_path = has_sub_path ? path_parts[1] : ""
 
+    alternative_cta = Maybe(ListingShape.where(community_id: @current_community.id).first)
```

Comment:
`ListingShape` model shouldn't be referenced directly from a controller. You can use the `ListingService::API::Api.shapes.get(community_id: @current_community.id)` method.